_rules/device-motion-user-interface-7677a9.md

```diff
@@ -0,0 +1,316 @@
+---
+id: 7677a9
+name: Device motion based functionality can also be operated from the user interface
+rule_type: atomic
+description: |
+  This rule checks that functionality that can be operated by device motion can also be operated by user interface components.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'    
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
+
+## Expectation 1
+
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, a set of one or more [instruments][instrument] is available supporting the same outcomes of the event.
```

Comment:
Got a few questions about this:
- "available": what does that mean? In the DOM? On another page? Anywhere on the web? As in, how do we ensure testers will do this consistently?
- "supporting the same outcomes" How do you know that something is the same outcome? Presumably this isn't just limited to "outcomes" that are apparent to the user, but things users might not be aware of too. What does it mean for the outcome of one event to be "the same" as that of another?

Reply:
- I've added to expectation 1 that the instrument should be located on the same web page. Since this instrument is supposed to offer the same functionality that is available through device motion, it does not make sense that it is located elsewhere (in this regard it is different from the instrument that is required to disable the device motion).
- Since this is an expectation and not the applicability, there is no need to objectively define "outcome" or "same". I agree that there might be some ambiguity regarding the scope of "outcomes". I've added a note clarifying that **all** outcomes resulting from the device motion should be considered. Given that you also expressed concerns about the ambiguity of "same" (I don't think it is ambiguous), I've replaced it with a synonym: indistinguishable.